flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -237,6 +237,13 @@ public ExecutionConfig getConfig() {
 		return cacheFile;
 	}
 
+	/**
+	 * Gets the config JobListeners.
+	 */
```

Comment:
Annotate as `@PublicEvolving`.

Reply:
thanks, but ExecutionEnvironment is for Dataset api not DatatStream Api.

Reply:
Yes, `ExecutionEnvironment` is for the `DataSet` API and there are also `JobListeners` there. So why not adding it?